primo-zip.py

```diff
@@ -41,19 +43,16 @@ def zip_up_source(zip_name, base_directory, directory):
 
 
 def main():
+    # user must input zip_name parameter at command line
+    if len(sys.argv) > 1:
```

Comment:
No change needed, but see also [argparse documentation](https://docs.python.org/3/library/argparse.html).  Since there's only one parameter currently, this code is OK (and simpler than `argparse`).